src/styles/Book.css

```diff
@@ -0,0 +1,100 @@
+.main-div {
+  display: grid;
+  grid-template-columns: 3fr 2fr 2fr;
+  padding-left: 1.288rem;
+  margin: 0.375rem 6.25rem 0.125rem 6.25rem;
+  background-color: #fff;
+}
+
+.title {
+  height: 1.813rem;
+  margin: 0.188rem 0 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+}
+
+.title,
+.btn,
+.author,
+.categories,
+.chapter {
+  font-weight: bold;
+}
+
+.btn,
+.author {
+  border: none;
+  color: #4386bf;
+  padding-left: 0;
+  font-size: 1rem;
+  margin-bottom: 20px;
+  padding-right: 40px;
+}
+
+.categories {
+  font-size: 0.875rem;
+  color: #888;
+}
+
+.btns {
+  margin-top: 30px;
+}
+
+.circle {
+  font-size: 40px;
+  margin: auto 0;
+}
+
+.round {
+  width: 4.25rem;
+  height: 4.25rem;
+  margin-right: 20px;
+  border: 6px solid;
+  border-color: #307bbe #307bbe #e8e8e8 #e8e8e8;
+  border-radius: 50%;
+}
+
+.logo-title,
+.user-icon {
+  color: rgb(12, 142, 250);
+}
+
+.user-icon {
+  margin-right: 6.5rem;
```

Comment:
- [ ] The user icon is still not aligned, please try using percentages or vw instead of rem.

![image](https://user-images.githubusercontent.com/38522578/140829683-4c7c68a9-0001-46ea-8fc5-ae21fdfa6d33.png)